.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
   - DJANGO='django<1.5'
   - DJANGO='django<1.6'
   - DJANGO='django<1.7'
+  - DJANGO='django<1.8'
+  - DJANGO='django<1.9'
```

Comment:
Yeah, sorry. You're right :)
Yes, ATM I vote for pinning 1.9.b1 instead of <1.10